org._2006Scape.Launcher.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org._2006Scape.Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.openjdk8
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=~/2006Scape:create
```

Comment:
I think you should use persist instead.

Reply:
We need manual access to ~/2006Scape to be able to edit the files inside & add Scripts used by the program